server/rest/dataset.py

```diff
@@ -393,7 +397,8 @@ def promoteItemToDataset(self, item, params):
                 minerva_metadata['original_files'] = [{
                     'name': file['name'], '_id': file['_id']}]
                 break
-            elif 'tif' in file['exts'] and file['mimeType'] == 'image/tiff':
+            elif (('tif' in file['exts'] or 'tiff' in file['exts'])
```

Comment:
Another way to write this which would extend to more extensions would be `{'tif', 'tiff'}.intersection(file['exts']) and file['mimeType'] == 'image/tiff')`.